soc/arm/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -24,9 +26,21 @@ config LOG_BACKEND_SWO_REF_FREQ_HZ
 endif # LOG_BACKEND_SWO
 
 # set the tick per sec as a divider of the LPTIM clock source
+if STM32_LPTIM_TIMER && STM32_LPTIM_CLOCK_LSE
 config SYS_CLOCK_TICKS_PER_SEC
-	default 4096 if STM32_LPTIM_TIMER && STM32_LPTIM_CLOCK_LSE
-	default 4000 if STM32_LPTIM_TIMER && STM32_LPTIM_CLOCK_LSI
+	default 4096 if "$(dt_node_int_prop_int,$(DT_STM32_LPTIM_PATH),clock-prescaler)" < 16
```

Comment:
Before changing the binding (which will have wide impacts) let's try to define a macro :
```
DT_ST_PRESCALER := st,prescaler
```
then use it as follows:
```
	default 4096 if "$(dt_node_int_prop_int,$(DT_STM32_LPTIM_PATH),$(DT_ST_PRESCALER))" < 16
```

Reply:
yes, exactly, now I have the macro ,-)